pom.xml

```diff
@@ -262,11 +262,11 @@
          -->
         <version.asciidoctor.plugin>2.2.2</version.asciidoctor.plugin>
         <version.org.jacoco>0.8.7</version.org.jacoco>
-        <version.org.jboss.galleon>5.0.3.Final</version.org.jboss.galleon>
+        <version.org.jboss.galleon>5.0.4.Final</version.org.jboss.galleon>
```

Comment:
5.0.5.Final has been released.

Reply:
@jfdenise Can that be handled via a follow-up PR? If this PR is ok as is it's better to get it in than to wait hours for CI.

Reply:
Hmm, but the Bootable jar test failure leads me to think it's not good as is.

Reply:
@bstansberry , thank-you for letting me now. I am going to investigate what is going on.

Reply:
We need to upgrade Bootable JAR maven plugin version. 7.0.0.Final depends on old Galleon. Will release it now. The bootable JAR upgrade should be done in this PR too. Will provide information there.

Reply:
@spyrkob , we need to upgrade to Bootable JAR 8.0.0.Alpha3. Just in case, change is done there: https://github.com/wildfly/wildfly/blob/main/pom.xml#L285
Thank-you.